Create 2025-Q1-BEAR-WG.md (#454)
ossf/tac: TI-reports/2025/2025-Q1-BEAR-WG.md

# 2025 Q1 BEAR WG

Chairs: [Marcela Melara](https://github.com/marcelamelara), [Yesenia Yser](https://github.com/Cyber-JiuJiteria) and [Jay White](https://github.com/camaleon2016)

## Overview

The BEAR WG (Formerly DEI WG) continues our mission to increase representation and strengthen
the overall effectiveness of the cybersecurity workforce. The WG continues to
see relatively low regular attendance in meetings (5 attendees on avg),
we are seeing increased engagement on Slack.

Over the past quarter, we've made progress in these areas:

* Committed to our social action mission we have rebranded the DEI WG to Belonging, Empowerment, Allyship, and Representation (BEAR) WG 
* Our monthly Community Office Hours have been regularly attracting 10+ attendees (on average).
* Our meeting and Office Hours recordings are available on YouTube, improving access.

> * We are in the planning process for a panel discussion at OSSNA '25.

Reviewer: ```suggestion
* We have submitted a proposal for a panel discussion on mentoring at OSS NA '25.
```

* We have submitted TI Finding Request for [LFX Mentorship Projects (summer cycle '25)](https://github.com/ossf/tac/issues/451)
* We are working through setting up what we call BEAR Hour. Details are still being worked through

> * SUbmitted Incubation Stage application which is still being updated. ( Name change is required on the application)

Reviewer: ```suggestion
* Submitted Incubation Stage application which is still being updated. ( Name change is required on the application)
```

## Community Office Hours

### Purpose

Our Community Office Hours are a monthly event in which a panel of industry
experts discuss a specific topic relevant to the WG's mission.
Each month, we aim to host diverse speakers that represent different
underrepresented groups, elevating different voices and connecting different
communities.

### Current Status

Since November, our Community Office Hours series has focused on topics especially
relevant to newcomers to OSS and the cybersecurity industry:

* Dec 5, 2024: [Standing out in the job market with your brand](https://youtu.be/smx2xnEuIFI?si=lEYsqUyq7-qVtxGk)
    - Speakers: Lexi Butler, Yesenia Yser

> * Jan 23: [Navigating OSS with Neurodiversity](TBD)

Reviewer: ```suggestion
* Jan 30: [Navigating OSS with Neurodiversity](https://www.youtube.com/watch?v=uOqoyLORzB4)
```

>     - Speakers: John Kjell, Jonathan Leitschuh Moderator: Marcela Melara

Reviewer: I think Yesenia moderated this one.
```suggestion
    - Speakers: John Kjell, Jonathan Leitschuh, Moderator: Yesenia Yser
```

### Up Next

> We are planning one more Community Office Hour before the end of 2024
> continuing our theme of newcomer-oriented topics:

Reviewer: This should be updated for 2025:
```suggestion
In 2025, we are planning our Community Office Hours around the theme of
community-oriented topics:
```

* Feb	Celebrating Black Excellence in OSS
* Mar	Mental Health Matters
* Apr	Celebrating Women in OSS

We will be [documenting Office Hour recaps](https://github.com/ossf/wg-dei/pull/9)
to enhance our YouTube videos and/or be submitted as blog posts.

### Questions/Issues for the TAC

We welcome topic and speaker suggestions for the remainder of the year!

## Panel Discussions at Major Conferences

### Purpose

Members of the OpenSSF BEAR WG community host panel discussions at major
industry conferences to raise awareness about the WG and to engage with
folks beyond the OpenSSF. A primary goal of the discussions is for
panelists to share their experiences and advice for newcomers to the
OSS and cybersecurity space.

### Upcoming Panel Discussions

We are looking forward to our panel discussion, titled 
"Finding Your Open Source Mentor: Strategies for Success" at OSSNA '25 
in June. Our goal for this panel discussion is to inspire and educate 
the audience as a range of OSS professionals share their experiences with 
mentoring from the perspective of both the mentee and the mentor.

#### Summary

> Panelists: Ijeoma Onwuka, Santiago Torres-Arias, Kairo de Araujo, Marcela Melara

Reviewer: Since I'm backup, I don't need to be listed here :)
```suggestion
Panelists: Ijeoma Onwuka, Santiago Torres-Arias, Kairo de Araujo
```

tech tip they would like to share with the OpenSSF and beyond.

While we had initially intended to begin this activity in August, the WG has
decided to delay this activity to 2025 due to a number of factors.

* **Mentorship**: Ahead of summer 2025, the WG is seeking collaboration with
other OpenSSF TIs to enroll in the [LFX mentorship program](https://lfx.linuxfoundation.org/tools/mentorship/). This will expose new and future
contributors to OpenSSF efforts while also advancing our efforts. As part of
this activity, the WG has applied for funds to sponsor these
mentorships.

* **Inclusive Contributing Guidelines**: There have been [a](https://github.com/ossf/tac/issues/330) [few](https://github.com/ossf/tac/issues/169) asks
to the TAC regarding the attraction and retention of maintainers. Working
together with other OpenSSF TIs, we aim to develop a small and approachable
set of contributing guidelines that focus on clear directions for project
newcomers and practices that facilitate different types of contributions.
The end goal is for our software and spec TIs to adopt these guidelines
as part of their lifecycle requirements.

### Questions/Issues for the TAC

> The BEAR WG intends to drive these activities, but need's the TAC's help in

Reviewer: ```suggestion
The BEAR WG intends to drive these activities, but needs the TAC's help in
```
